src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -90,72 +90,89 @@ class oopDesc;
 // doesn't provide appropriate definitions, they should be provided in
 // the compiler-specific definitions file (e.g., globalDefinitions_gcc.hpp)
 
-#define BOOL_TO_STR(_b_) ((_b_) ? "true" : "false")
+// Guide to the suffixes used in the format specifiers for integers:
+//        - print the decimal value:                   745565
+//  _X    - print as hexadecimal, without leading 0s: 0x12345
+//  _X_0  - print as hexadecimal, with leading 0s: 0x00012345
+//  _H    - print as hexadecimal, without 0x prefix
```

Comment:
I think we don't have any of these "H" suffixed names.

Reply:
The patch has one usage of SIZE_FORMAT_H_W. We could of course just inline that usage, if you think that makes more sense.

Reply:
I don't see a use, or even a definition, of that name.  There was the INTPTR_FORMAT_H_W that was moved to Shenandoah code.

Reply:
Oh, it looks like you removed SIZE_FORMAT_H_W.  I guess I missed it while it was there.